airbyte-integrations/connectors/source-hubspot/source_hubspot/streams.py

```diff
@@ -891,7 +891,7 @@ def filter_by_state(self, stream_state: Mapping[str, Any] = None, record: Mappin
         # save the state
         self.state = {self.cursor_field: int(max_state) if int_field_type else max_state}
         # emmit record if it has bigger cursor value compare to the state (`True` only)
-        return record_value > state_value
+        return record_value >= state_value
```

Comment:
Very edge case but we had a similar discussion for Salesforce [here](https://github.com/airbytehq/airbyte/pull/34107#issuecomment-1892437936)